src/utils/helpers/browser-type-check/browser-type-check.js

```diff
@@ -0,0 +1,5 @@
+function browserTypeCheck(_window) {
+  return Boolean(_window.chrome || _window.sidebar);
```

Comment:
I think a comment in the code would be helpful here, to explain exactly what this is checking, ideally including a URL of some online documentation for reference, eg:
```js
/**
 * Check for Chrome or Firefox.
 * @see https://stackoverflow.com/q/.....
 * @see https://developer.mozilla.org/.....
 */
```

Reply:
Yeah very good point will add now

Reply:
Thanks :white_check_mark: